src/views/accounts/Integration/ApplicationList.vue

```diff
@@ -55,7 +55,7 @@ export default {
             formatter: DetailFormatter
           },
           secret: {
-            label: this.$t('Secret'),
+            label: 'Secret',
             formatter: CopyableFormatter,
             formatterArgs: {
               shadow: true,
```

Comment:
I cannot analyze the provided code since I am unable to view or read external files. Please share this information or paste it here so I can assist you further.